homework3/semi-uber/Server/api/handlers/trucksHandlers/trucksPatchHandler.js

```diff
@@ -0,0 +1,33 @@
+const log4js = require("log4js");
+const logger = log4js.getLogger();
+const errorHandler = require("../error.handler");
+const { Truck } = require("../../Schemas/truck.schema");
+
+const trucksPatchHandler = async (req, res) => {
+  const { userID, truckID } = req.body;
+
+  try {
+    const clearAssign = await Truck.updateMany(
+      { created_by: userID },
+      { $set: { assigned_to: null } }
+    );
+  } catch (error) {
+    return errorHandler(error.message, res);
+  }
+  logger.info("All trucks is not assigned");
+  try {
+    const assignedTruck = await Truck.findByIdAndUpdate(truckID, {
```

Comment:
The same, you do not to write it to variable, `await` and function call will be enough

Reply:
Done